tldw_Server_API/app/core/DB_Management/migration_tools.py

```
@@ -89,17 +89,19 @@ def migrate_sqlite_to_postgres(
     if not sqlite_path.exists():
         raise FileNotFoundError(f'SQLite database not found: {sqlite_path}')
 
-    logger.info("Starting migration of %s database from file %s", label, sqlite_path.name)
+    logger.info(f"Starting migration of {label} database")
```

Ruff (0.15.9)

[warning] Logging statement uses f-string

(G004)